sdk/communication/Azure.Communication.CallAutomation/src/CallMedia.cs

```diff
@@ -494,6 +495,7 @@ public virtual Response<HoldResult> Unhold(UnholdOptions options, CancellationTo
             try
             {
                 var request = new UnholdRequestInternal(CommunicationIdentifierSerializer.Serialize(options.TargetParticipant));
+                request.OperationContext = options.OperationContext;
```

Comment:
*NIT*: `UnholdRequestInternal` has constructor for `operationContext`
```suggestion
 new UnholdRequestInternal(CommunicationIdentifierSerializer.Serialize(options.TargetParticipant), options.OperationContext);
```

Reply:
Used the constructor for operation context.